src/devices/QwiicButton/QwiicButton.QueueManipulation.cs

```diff
@@ -0,0 +1,119 @@
+﻿//// Licensed to the .NET Foundation under one or more agreements.
+//// The .NET Foundation licenses this file to you under the MIT license.
+//// See the LICENSE file in the project root for more information.
+
+using Iot.Device.QwiicButton.RegisterMapping;
+
+namespace Iot.Device.QwiicButton
+{
+    public sealed partial class QwiicButton
+    {
+        /// <summary>
+        /// Returns whether the queue of button press timestamps is full.
+        /// </summary>
```

Comment:
please add comment to mention what happens when queue is full and you get an event? Are you gonna lose new events or some error condition will happen?

Reply:
Interesting but I don't know - Sparkfun hasn't documented it anywhere and I don't intend on reading the firmware :-)